docs/kubernetes/README.md

```diff
@@ -3,15 +3,30 @@
 
 ### Prerequisite Checklist
 
+- `kubectl` installed and configured with authentication to the cluster.
 - Dedicated Namespace to run the jobs.
 - RWX volume to store the evaluation results (check `volume.yml` for inspiration).
 
-### Running a job
+### Running a evaluation without the eval-dev-quality kubernetes runtime
 
 - Change the `command` in `job.yml` to the desired command.
 - Run it with `kubectl --namespace $NAMESPACE apply -f job.yml`.
 - Check the job with `kubectl get pods --namespace $NAMESPACE` until status shows `completed`.
 - Remove the job with `kubectl --namespace $NAMESPACE delete --force -f job.yml`.
+- [Getting the evaluation data](#getting-the-evaluation-data)
+
+### Running multiple evaluations with the eval-dev-quality kubernetes runtime
+
+- Define all the models with `--model` which should be run inside the containerized workload.
+- Define the parameter `--runtime kubernetes` to indicate that jobs should run inside a kubernetes cluster.
+- Define the parameter `--parallel 20` to indicate how many jobs should run simultaneously.
+- [Getting the evaluation data](#getting-the-evaluation-data)
```

Comment:
Would be amazing if the runtime would automatically fetch the results. Cause from the steps below that seems like something that can be automated. Can you look into that and note it down as a follow-up?

Reply:
https://github.com/symflower/eval-dev-quality/issues/198 - Iteration 4

But it has to be optional because I am unable to determine the way other people have their kubernetes cluster defined for Multi read write volumes.